### ISSUE TITLE
@graphql-codegen/flow with `useFlowExactObjects: true` creates invalid javascript for enums

### ISSUE THREAD

**Issue by philiiiiiipp** [dotansimha/graphql-code-generator#1634]
**Describe the bug**
 

**To Reproduce**

1. My GraphQL schema:

```graphql
enum Example {
  Test1
  Test2
}
```

3. My `codegen.yml` config file:

```yml
schema: ./test.gql
generates:
  ./types.flow.js:
    plugins:
      - flow
    config:
      useFlowExactObjects: true
```

If you run the codegen here it will produce

```
/* @flow */



/** All built-in and custom scalars, mapped to their actual values */
export type Scalars = {|
  ID: string,
  String: string,
  Boolean: boolean,
  Int: number,
  Float: number,
  AWSDateTime: string,
  AWSJSON: string,
|};

export const ExampleValues = Object.freeze({|
  Test1: 'Test1', 
  Test2: 'Test2'
|});


export type Example = $Values<typeof ExampleValues>;
```
where `ExampleValues` should not contain the `|`.

**Comment by dotansimha** [comment]
Thanks @philiiiiiipp , fixed in: https://github.com/dotansimha/graphql-code-generator/pull/1663

**Comment by dotansimha** [comment]
Fixed in 1.0.7 :)
